### PR TITLE
New package: Microsoft.Promptflow version 1.7.0

### PR DESCRIPTION
**Pull request by YingChen1996**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/148162)

### CODE REVIEW

File: manifests/m/Microsoft/Promptflow/1.7.0/Microsoft.Promptflow.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.Promptflow
```
Comment (YingChen1996) [inline manifests/m/Microsoft/Promptflow/1.7.0/Microsoft.Promptflow.installer.yaml:4]: fixes #148166

**Comment by YingChen1996** [comment]
fixes https://github.com/microsoft/winget-pkgs/issues/148166

**Comment by YingChen1996** [comment]
Not sure why have this alert when testing locally
image

**Comment by stephengillie** [comment]
Hi @YingChen1996, 

> Not sure why have this alert when testing locally image

This warning is common for new software packages. Our pipelines are set up so that these are skipped or bypassed, but other users might still see it. I believe there might be a site where packages can have ratings added, which might eventually remove this warning.

**Comment by stephengillie** [comment]
Verify: 
image

**Review by stephengillie — APPROVED** [review]

**Comment by Trenly** [comment]
> Verify: image

@stephengillie - The entry you show in this screenshot does not exactly match the `DisplayName` specified in the manifest. This will cause issues where the CLI won't be able to correlate this program with certainty

**Pull request merged**